tests/assets/default.conf

```diff
@@ -35,3 +35,5 @@ propagation-error-behavior panic
 
 # Make sure shutdown doesn't fail if there's an initial AOFRW
 shutdown-on-sigterm force
+
+enable-debug-assert yes
```

Comment:
This isn't sufficient, this is only used for normal tests but it won't cover the cluster tests. Maybe also update `init-tests.tcl`.